config/runner.json

```diff
@@ -0,0 +1,10 @@
+{
+  "suites": {
+    "eddsa-rdfc-2022": {
+      "tags": ["eddsa-rdfc-2022"]
+    },
+    "eddsa-jcs-2022": {
+      "tags": ["eddsa-jcs-2022"]
+    }
+  }
+}
```

Comment:
@aljones15 if the tag is identical to the suite name, why is this mapping document needed?